Formula/gdbm.rb

```diff
@@ -50,20 +51,129 @@ def install
 end
 
 __END__
---- src/gdbmshell.c.orig
+commit 32517af75ac8c32b3ff4870e14ff28418696c554
```

Comment:
Thanks, @FnControlOption. Do you mind submitting this patch to Homebrew/formula-patches?

Reply:
https://github.com/Homebrew/formula-patches/pull/396